engine/src/flutter/shell/platform/darwin/macos/BUILD.gn

```diff
@@ -18,6 +18,31 @@ shell_gpu_configuration("macos_gpu_configuration") {
   enable_metal = true
 }
 
+config("config_macos") {
+  include_dirs = [ target_gen_dir ]
+  lib_dirs = mac_swift_lib_paths
+
+  # Allow use of @testable imports in debug builds and when tests are enabled.
+  if (enable_ios_unittests) {
+    swiftflags = [ "-enable-testing" ]
+  }
+}
+
+config("config_macos_test") {
```

Comment:
Is this config used anywhere?

Reply:
It's not currently used, but will be needed for Swift XCTest targets. Sadly, all our unit test targets are GoogleTest (C++) today -- even the very few Obj-C ones that happen to use XCTest -- so I'm going to need to write some GoogleTest/XCTest adaptor code first.

In the ideal world, I'd have written that first, but like the iOS tests, it's still pretty useful to lock in the Swift stuff quickly, then build on it in a followup patch.

In the even more ideal world, the embedder would use XCTest rather than GoogleTest, since embedders should really live in a more platform-specific world, but that's a longer-term goal, maybe.